Computer History Museum Archive.js

```diff
@@ -0,0 +1,589 @@
+						"mimeType": "application/pdf",
+						"title": "Knuth, Donald oral history"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/catalog/102646282",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "IBM 1401 Programming Systems",
+				"creators": [],
+				"date": "1959",
+				"abstractNote": "Subject: 1401 data processing system (Computer); promotional materials; Digital computer: mainframe; 1401 programming systems (Software); COBOL (Software); Business applications; Scientific applications; Software; Digital communications--Social aspects; International Business Machines Corporation (IBM). Data Processing Division\nCategory: Promotional Material\nDescription: The brochure explains the IBM 1401 programming languages and their application to the 1401 data processing system. The brochure is printed in black, white, and blue. The front cover shows the words Programming and Systems in a repetitive design with the name Donald G. McBrien stamped in the upper right corner. The back cover shows the company logo on a blue background. Throughout the inside pages are black and white photographs of the computer and images of reports generated by the system. Text contents include: What is a 1401 program?; What is a stored program machine?; What are 1401 programming systems?; What 1401 programming systems mean to management?; IBM programming systems; Here's how one of the 1401 programming systems -- Report Program Generator -- works to increase programming efficiency; New IBM services include:; Other services available to every IBM customer.\nType: Document",
+				"archive": "Computer History Museum",
+				"archiveLocation": "X3067.2005",
+				"extra": "Identifying Numbers: Other number: 520-1368\nDimensions: 9 5/8 x 7 6/8 in.\nCopyright Holder: International Business Machines Corporation (IBM). Data Processing Division",
+				"libraryCatalog": "102646282",
+				"numPages": "6",
+				"publisher": "International Business Machines Corporation. Data Processing Division. (IBM); U.S.",
+				"attachments": [
+					{
+						"mimeType": "application/pdf",
+						"title": "IBM 1401 Programming Systems"
+					},
+					{
+						"title": "IBM 1401 Programming Systems Image 1",
+						"mimeType": "image/png"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/catalog/102623002",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "A guide to Fortran IV programming",
+				"creators": [],
+				"date": "1972",
+				"abstractNote": "Category: Book\nDescription: Second edition.  Signed by McCracken on title page.\nType: Document",
+				"archive": "Computer History Museum",
+				"archiveLocation": "X3682.2007",
+				"extra": "Identifying Numbers: ISBN10: 0471582816\nLCCN\n72-4745\nLOC call num\nQA76.73.F25 M3 1972\nDimensions: 28 cm.",
```

Comment:
- We should parse out the things we understand. E.g., the ISBN should go in the ISBN field, and we have a Call Number field.
- Extra values should generally be `key: val`, so having "Identifying Numbers:" and then additional lines isn't ideal. It would be better to parse this if possible and create separate `key: val` lines.

Reply:
fixed